qa/os/centos-7/build.gradle

```diff
@@ -1 +0,0 @@
-project.ext.shouldTestDocker = true
```

Comment:
Why has this been removed?

Reply:
It was unintentional, I've added it back